_templates/header.php

```diff
@@ -77,6 +77,11 @@
         <link rel="stylesheet" type="text/css" media="all" href="https://fonts.googleapis.com/css?family=Raleway:100|Open+Sans:300,400,600|Droid+Sans+Mono">
         <link rel="stylesheet" type="text/css" media="all" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
         <link rel="stylesheet" type="text/css" media="all" href="styles/main.css">
+        
+        <script>
+            (function(d,s,f){g=d.createElement(s),u=d.getElementsByTagName(s)[0],g.async=1,g.src=f,u.parentNode.insertBefore(g,u)})
+        	(document,'script','https://beta.elementary.io/branch/hsts-conditional/backend/hsts.php')
```

Comment:
Yes you can't, but I said dynamic. This is a php file, it's possible to determine the needed path. If only there is a check for `$_SERVER['HTTP_HOST']` Maybe also some looking in `$_SERVER['REQUEST_URI']` to get the branch path.

This is probably beter served as a function if it not already exists